packages/camera/camera_platform_interface/lib/src/types/camera_event.dart

```diff
@@ -0,0 +1,76 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+
+import 'package:meta/meta.dart';
+
+/// Event emitted from the platform implementation.
+class CameraEvent {
+  /// Creates an instance of [CameraEvent].
+  ///
+  /// The [eventType] argument is required.
+  ///
+  /// Depending on the [eventType], the [captureSize], [previewSize] and
+  /// [errorDescription] arguments can be null.
+  CameraEvent({
```

Comment:
It is normally useful that these events contain the "cameraId" or "textureId" of the camera that originated them; that way event objects can be passed around in the plugin/app and still be linked to the camera that originated them later (imagine, a top-level error handler that disposes of a camera, or similar)

Reply:
All the events now have the cameraId added to them